src/command/decorator.ts

```diff
@@ -70,6 +71,7 @@ export function command(
                 optional: optionals.includes(i + 1)
             })),
             single: opts.single || false,
+            exactArgs: opts.exactArgs || false,
```

Comment:
(re: PR description) I think this is fine since:
- We preserve backwards compatibility
- Users may want to add a "comment" at the end of their command, e.g. a ban command with just one argument (user) being ran like `!ban @evil-person if anyone reads this later, I banned them because foo!`.

Reply:
Gotcha, that makes perfect sense.